v1.1.yml

```diff
@@ -1591,3 +1767,259 @@ definitions:
       email:
         type: string
         x-nullable: true
+
+  StudentObject:
```

Comment:
What exactly is the purpose of wrapping the Student, Teacher, etc. definitions in objects? I'm assuming that this has something to do with the polymorphism of events.